index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>We Care</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+
+<body>
+  <section class="main-container">
+    <header>
+      <div class="account">
+        <img src="images/icons/facebook-icon.svg" alt="facebook icon">
+        <img src="images/icons/twitter-icon.svg" alt="twitter icon">
+        <a href="#">English</a>
+        <a href="#">My Page</a>
+        <a href="#">Logout</a>
+      </div>
+      <nav>
+        <i class="material-symbols-outlined menu">
+          menu
+        </i>
```

Comment:
- [ ] I would suggest wrapping the hamburger icon in a link tag `<a>` for semantic purpose.  The semantic HTML tags help the search engines and other user devices to determine the importance and context of web pages.